### PR TITLE
Timelocal

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
Added support to Appliance Time and Locale Configuration.
(pending) 
--> initialize go.mod and go.sum

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
 
 | Endpoints                                                                       | Verb     | V200 | V300 | V500 |V600 |V800 |V1000| V1200 | V1600 | V1800 | V2000 | V2200 | V2400 | V2600 | V2800 | V3000 |
 | --------------------------------------------------------------------------------------- | -------- | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: |
+|     **Appliance Time and Locale Configuration** |
```
Comment (yuvirani) [inline endpoints-support.md:23]: Move below snmp

File: oneview/resource_appliance_time_and_locale_configuration_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+				Config: testAccTimeandLocaleConfiguration,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTimeandLocaleConfigurationExists(
+						"oneview_appliance_time_and_locale.test", &TimeandLocaleConfiguration),
+					resource.TestCheckResourceAttr(
+						"oneview_appliance_time_and_locale.test", "locale", "test_locale",
+					),
+					resource.TestCheckResourceAttr(
+						"oneview_appliance_time_and_locale.test", "timezone", "test_zone",
+					),
```
Comment (yuvirani) [inline oneview/resource_appliance_time_and_locale_configuration_test.go:41]: Add testCheck for ntpServers and data_time

File: oneview/data_source_appliance_time_and_locale_configuration.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+			"locale_displayname": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ntp_servers": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Set:      schema.HashString,
```
Comment (yuvirani) [inline oneview/data_source_appliance_time_and_locale_configuration.go:44]: Whats the use of this Set ?

**Review by yuvirani — APPROVED** [review]

File: oneview/resource_appliance_time_and_locale_configuration.go [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+				Computed: true,
+				Optional: true,
+			},
+			"ntp_servers": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required: true,
+				ForceNew: true,
```
Comment (AsisBagga) [inline oneview/resource_appliance_time_and_locale_configuration.go:55]: any specific reason why forceNew?

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,7 +13,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 1. Added support to Connection Templates Resource.
    - GET /rest/connection-templates/{id}
    - PUT /rest/connection-templates/{id}
-2. Added support to Appliance SNMPv1 Trap Destinations
+2. Added support to Appliance Time and Locale Configuration.
```
Comment (VenkateshRavula) [inline CHANGELOG.md:16]: Add this in supported features list

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**